### DISCUSSION TITLE
Easily enable/disable Line Suggestions

### DISCUSSION THREAD

**Discussion by vieiraes** [community/community#15065] [Copilot Conversations]
A very good feature, but sometimes I need to disable and enable later. A button must be nice

**Comment by jamesonknutson** [comment]
The setting `github.copilot.inlineSuggest.enable` determines whether or not Copilot will provide you with inline suggestions in the Editor. You could bind that to a keyboard shortcut, [or if you're on VS Code and prefer something more out-of-the-way, you could use my extension that disables it automatically within the specified textmate scopes (e.g. comments)](https://github.com/jamesonknutson/disable-copilot-comment-completions).
